include/riak_kv_index.hrl

```diff
@@ -45,4 +45,30 @@
           max_results :: integer() | undefined
          }).
 
--define(KV_INDEX_Q, #riak_kv_index_v3).
+-record(colocation, {
```

Comment:
I agree - I couldn't think of the right name and Russell and I chewed it over but didn't come to any conclusion - I think its wrong but would like a consensus on what right is...